### PR TITLE
38

### PR DESCRIPTION
**Pull request by Dcolonel6**
# Description

This pull request is part creating a reusable partner orgs section. We worked on this with @nziokikevin

Example
This pull request is part of the work to make it easier for people to contribute to naming convention guides. 
One of the easiest way to make small changes would be using the Edit on Github button.

To achieve this, we needed to:

```
- Find the best Gitbook plugin which can do the work
- Integrate it in all the pages to redirect the user to the right page on GitHub for editing
- Make it visible on the page so users can notice it easily
```

Fixes # 38 - *Create Reusable Partner Orgs Section*: [Fixes # 38](https://github.com/italanta/elewa-group/issues/38)
## Type of change
- [X] New feature (non-breaking change which adds functionality)

# Screenshoots

## Large screens
Screenshot from 2023-02-10 11-38-17

## Mobile Screen
Screenshot from 2023-02-10 11-42-01

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. 
Please also list any relevant details for your test configuration if necessary.
- [x] Test A
- [X] Test B

# Checklist:
- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by Dcolonel6** [comment]
Please review @IanOdhiambo9

File: apps/elewa-group-website/src/app/app.component.html [hunk 1]
```diff
@@ -1 +1 @@
-<elewa-group-home-page></elewa-group-home-page>
\ No newline at end of file
+<elewa-group-elewa-group-horizontal-list-orgs></elewa-group-elewa-group-horizontal-list-orgs>
```
Comment (IanOdhiambo9) [inline apps/elewa-group-website/src/app/app.component.html:1]: Only the homepage should be here, remove your tag after local testing

File: apps/elewa-group-website/src/app/app.module.ts [hunk 2]
```diff
@@ -9,6 +9,10 @@ import { HomePageModule } from '@elewa-group/pages/elewa/home';
 
 import { CardsModule } from '@elewa-group/features/components/cards';
 import { LayoutModule } from '@elewa-group/elements/layout';
+import { UiListsModule } from '@elewa-group/features/components/ui-lists';
+
+
+import { ButtonsModule } from '@elewa-group/features/components/buttons';
```
Comment (IanOdhiambo9) [inline apps/elewa-group-website/src/app/app.module.ts:16]: Not needed here

File: libs/features/components/buttons/project.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        }
+      }
+    },
+    "lint": {
+      "executor": "@nrwl/linter:eslint",
+      "outputs": ["{options.outputFile}"],
+      "options": {
+        "lintFilePatterns": [
+          "libs/features/components/buttons/**/*.ts",
+          "libs/features/components/buttons/**/*.html"
```
Comment (IanOdhiambo9) [inline libs/features/components/buttons/project.json:28]: This is not part of this issue

File: libs/features/components/ui-lists/src/lib/components/elewa-group-horizontal-list-orgs/elewa-group-horizontal-list-orgs.component.html [hunk 4]
```diff
@@ -0,0 +1,5 @@
+<ul class="horizontal-list">
+    <li class="list-logo" *ngFor="let logo of logos">
+      <img class="logo" src={{logo.src}} alt={{logo.alt}} srcset={{joinSrcSet(logo.srcset)}}/>
+    </li>
+</ul>
```
Comment (IanOdhiambo9) [inline libs/features/components/ui-lists/src/lib/components/elewa-group-horizontal-list-orgs/elewa-group-horizontal-list-orgs.component.html:5]: Make the logos a pair of two on each row in the mobile view.

File: libs/features/components/ui-lists/src/lib/components/elewa-group-horizontal-list-orgs/elewa-group-horizontal-list-orgs.component.ts [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+export class ElewaGroupHorizontalListOrgsComponent {
+  @Input() logos = [
+    {
+      src: "https://rwanda.vvob.org/sites/rwanda/files/styles/780pxx330px/public/images/mainimagebig/ok.jpg?itok=zMR2_g-z",
+      alt: "rwanda vvob logo",
+      srcset: [
+        
+        "https://rwanda.vvob.org/sites/rwanda/files/styles/200px/public/logo-white_1_0.png?itok=lEAU5wt_ 240w",
+        "https://pbs.twimg.com/profile_images/1333450887080112131/y17zlw7O_400x400.jpg 300w",
+        "https://rwanda.vvob.org/sites/rwanda/files/styles/780pxx330px/public/images/mainimagebig/ok.jpg?itok=zMR2_g-z 720w"
```
Comment (IanOdhiambo9) [inline libs/features/components/ui-lists/src/lib/components/elewa-group-horizontal-list-orgs/elewa-group-horizontal-list-orgs.component.ts:17]: The logos will be relatively small in size, just use a list of urls instead.

**Review by IanOdhiambo9 — CHANGES_REQUESTED** [review]
Elewa group button is not part of this issue. sync fork.

**Comment by IanOdhiambo9** [comment]
Resolve conflicts on branch

**Pull request closed without merging**